bundler.d/development.rb

```diff
@@ -12,4 +12,5 @@
 
   gem 'bullet'
   gem "parallel_tests"
+  gem 'jasmine-rails'
```

Comment:
Apparently the `rake spec:javascript` task to run Jasmine won't run if I don't put it here too. You can also test by going to localhost:3000/specs